day-3/src/App.js

```diff
@@ -0,0 +1,95 @@
+import React, { Component } from 'react';
+import './App.css';
+// import injectTapEventPLugin from 'react-tap-event-plugin';
+import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider';
+import AppBar from 'material-ui/AppBar';
+
+var quotes = ["Just hire a lawyer", "Empower your team", "Raise your prices", "Ignore most of the advice you're given", "Stop trying to hire a unicorn"];
+
+
+
+class App extends Component {
+
+  constructor (props) {
+    super (props);
+    this.state = {
+      currentIndex: 0,
+      count: 0,
+    };
+  }
+
+  clickRandom() {
+    this.setState({
+      currentIndex: Math.floor(Math.random() * 4),
+      count: this.count + 1, 
+    })
+
+  }
+
+
+  clickThrough() {
+    if ( this.state.currentIndex === 4) {
+        this.setState( {
+          attributes: quotes, 
+          currentIndex: 0,
+          count: this.count + 1, 
+        })
+      }
+      else {
+        this.setState( {
+          currentIndex: this.state.currentIndex +1,
+          count: this.count + 1, 
+        }) 
+      }
+  }
+
+  rotateQuotes() {
+      if ( this.state.currentIndex === 4) {
+        this.setState( {
+          currentIndex: 0,
+          count: this.count + 1, 
+        })
+      }
+      else {
+        this.setState( {
+          currentIndex: this.state.currentIndex +1,
+          count: this.count + 1, 
+        }) 
+      }
+  }
+  
+
+  render() {
+    return (
+      <div className="App">
+        <div className="App-header">
+          <h2>Great Startup Advice</h2> 
+          <MuiThemeProvider><AppBar title="Header" iconClassNameRight="muidocs-icon-navigation-expand-more"/></MuiThemeProvider>
```

Comment:
`MuiThemeProvider` should be a top level dom element. it should not wrap each Material UI component but rather the entire application